superset-frontend/src/dashboard/util/getDashboardFilterKey.ts

```diff
@@ -16,11 +16,20 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-export function getDashboardFilterKey({ chartId, column }) {
+
+interface GetDashboardFilterKeyProps {
+  chartId: string;
+  column: number;
```

Comment:
`column` should be string, mostly is column name to be filtered on

Reply:
updated